webapp/package.json

```diff
@@ -9,6 +9,7 @@
 		"watchdev": "cross-env NODE_ENV=dev webpack --watch --progress --config webpack.dev.js",
 		"deveditor": "cross-env NODE_ENV=dev webpack server --config webpack.editor.js",
 		"test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
I am not sure why we need this?

Reply:
Also adding a command at the workspace would mean that supporting the same in all the package I suggest you to please remove this change

Reply:
it's for convenience so contributors can have their tests run when making changes without having to type "npm run test". Also different IDEs support running scripts by clicking on them which is nice to have.

I removed the script since it's not strictly needed.